atomic_reactor/inner.py

```diff
@@ -325,14 +330,28 @@ def build_docker_image(self):
                 self.autorebuild_canceled = True
                 raise
 
+            start_time = datetime.datetime.now()
+            self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
+
             build_result = self.builder.build()
+
+            try:
+                finish_time = datetime.datetime.now()
+                duration = finish_time - start_time
+                seconds = duration.total_seconds()
+                logger.debug("build finished in %ds", seconds)
+                self.plugins_durations['dockerbuild'] = seconds
+            except Exception:
+                logger.debug(traceback.format_exc())
+
             self.build_logs = build_result.logs
 
             self.build_failed = build_result.is_failed()
 
             if build_result.is_failed():
                 # The docker build failed. Finish here, just run the
                 # exit plugins (from the 'finally:' block below).
+                self.plugins_errors['build'] = ''
```

Comment:
Agreed, lets make it consistent to ```dockerbuild```

We set it to an empty string as its complicated to get an actual error message without pasting the whole docker log output